integration-tests/kogito/javax.inject.Named

```diff
@@ -0,0 +1,4 @@
+org.drools.project.model.SessionRuleUnit_canDrinkKS
+org.drools.project.model.SessionRuleUnit_simpleKS
+io.quarkus.it.kogito.jbpm.OrdersProcess
+io.quarkus.it.kogito.jbpm.OrderItemsProcess
```

Comment:
Hmmm, this shouldn't have been committed?

Is it an old branch? Because the current master shouldn't generate this file anymore.

Reply:
Oh, that's weird.  I just rebased on master.  I'll fix it in a moment.